packages/core/src/dialects/abstract/query-interface.js

```diff
@@ -248,6 +188,26 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
     return out.length === 1;
   }
 
+  /**
```

Comment:
Bit nitpicky but can we keep this in its original place?

Reply:
Done